doc/classes/SubViewportContainer.xml

```diff
@@ -4,7 +4,7 @@
 		A container used for displaying the contents of a [SubViewport].
 	</brief_description>
 	<description>
-		A container that displays the contents of underlying [SubViewport] child nodes. It uses the combined size of the [SubViewport]s as minimum size, unless [member stretch] is enabled.
+		An interface that displays the contents of underlying [SubViewport] child nodes and handles input event propagation. It uses the combined size of the [SubViewport]s as minimum size, unless [member stretch] is enabled.
```

Comment:
What's the reasoning for saying "An interface". It's "a container" as in it extends the **Container** class.

Reply:
I had two reasons for this change:
1. Besides simply containing and displaying viewports, the class `SubViewportContainer` additionally propagates notifications and input events, so it does more, than the other **Container** classes.
2. The **Container** class also doesn't really fit for the functionality of `SubViewportContainer` (See https://github.com/godotengine/godot-proposals/issues/7081), since its intention is to hold Control nodes and not Viewport nodes.

Let me know, if I should make adjustments.